libsplinter/src/biome/users/mod.rs

```diff
@@ -13,3 +13,32 @@
 // limitations under the License.
 
 pub mod database;
+
+use database::models::UserModel;
+
+pub struct SplinterUser {
+    id: String,
```

Comment:
Do we need some sort of metadata here to handle the display name and other fields used by apps?

Reply:
I thought that kind of stuff would be implemented once we had user preferences/profile support in libsplinter? I think this kind of information should be separate from user, so we can keep it all modular in optional features.

Reply:
That's true - we might not want that as a specific field in the table. What do you think about implementing a metadata table for users (similar to how we were talking about implementing it for notifications)?

Reply:
That is certainly an option! Though, I feel it deserves a lengthier discussion outside the scope of this PR? I feel we need more clarity on how we want these application dependent user preferences fields to be managed